brew_wizard_ui/src/app/app.component.scss

```diff
@@ -3,3 +3,9 @@
   margin-bottom: 60px;
   min-height: calc(100vh - 120px);
 }
+
+@media screen and (max-width: 420px) {
```

Comment:
For bootstrap the screen breakpoint is 576

Reply:
See if this works btw:
// Example usage:
@include media-breakpoint-down(sm) {
  .body {
    padding-top: 80px;
  }
}